core/client/src/main/java/alluxio/client/file/options/DeleteOptions.java

```diff
@@ -24,6 +25,7 @@
 @NotThreadSafe
 public final class DeleteOptions {
   private boolean mRecursive;
+  private boolean mRemoveUFSFile;
```

Comment:
the command that fixes the consistency could talk to directly to UFS instead of asking Alluxio to delete the file on our behalf